elizhahn/news-box
# Feature/display default articles

## What is being added/changed?
- The application on load will show a default list of NYT's top home articles
- CardList component was created to host ArticleCards
- ArticleCard component was created to render each article title, publish date, and a button that will be used to see more details
- header was given basic styling
## How should this be tested?
- Visit localhost: 3000 and you should see a list of unique cards displaying a title, date, and button

## future iterations?
- functionality will be added to the card to render more details
- additional styling
## Linked Issues?
- #7
